js/src/components/google-ads-account-card/claim-account-button.js

```diff
@@ -48,6 +53,7 @@ const ClaimAccountButton = ( { onClick = noop, ...restProps } ) => {
 			eventName="gla_open_ads_account_claim_invitation_button_click"
 			eventProps={ getEventProps() }
 			onClick={ handleClaimAccountClick }
+			loading={ ! inviteLink || loading }
```

Comment:
Why is checking the `inviteLink` now necessary?

Reply:
@asvinb I think `loading` should be `true` in such cases where the data resolution is not completed yet.
CC: @joemcgill